client/src/views/HomeView.tsx

```diff
@@ -259,6 +265,11 @@ const HomeView = () => {
                 onChange={changeName}
                 value={name}
               />
+              <Tooltip title={!fixBruteAppearance ? t('lockBruteAppearance') : t('unlockBruteAppearance')}>
+                <IconButton onClick={() => setFixBruteAppearance(!fixBruteAppearance)} size="small" sx={{ float: 'right' }}>
```

Comment:
Always use callbacks on react setters that don't depend on anything else
```suggestion
                <IconButton onClick={() => setFixBruteAppearance((prev) => !prev)} size="small" sx={{ float: 'right' }}>
```

Reply:
I did the call back but I don't understand the (prev) => !prev) part

Reply:
It uses the previous state as reference. See https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state